include/gridtools/common/cuda_util.hpp

```diff
@@ -53,21 +53,21 @@ namespace gridtools {
             return unique_cuda_ptr<Arr>{ptr};
         }
 
-        template <class T, std::enable_if_t<!std::is_array<T>::value, int> = 0>
+        template <class T, std::enable_if_t<!std::is_array_v<T>, int> = 0>
         unique_cuda_ptr<T> cuda_malloc() {
             T *ptr;
             GT_CUDA_CHECK(cudaMalloc(&ptr, sizeof(T)));
             return unique_cuda_ptr<T>{ptr};
         }
 
-        template <class T, std::enable_if_t<std::is_trivially_copyable<T>::value, int> = 0>
+        template <class T, std::enable_if_t<std::is_trivially_copy_constructible_v<T>, int> = 0>
```

Comment:
Is the change from `is_trivially_copyable` to `is_trivially_copy_constructible` intended?

Reply:
yes. explained in PR description

Reply:
Ups, too boring review, I even missed the description. Sorry!